tools/starhistory/github.go

```
@@ -0,0 +1,224 @@
+			return nil, "", fmt.Errorf("create GitHub request: %w", err)
+		}
+		req.Header.Set("Accept", githubStarMediaType)
+		req.Header.Set("Authorization", "Bearer "+token)
+		req.Header.Set("User-Agent", "wails-star-history")
+
+		resp, err := client.Do(req)
+		if err != nil {
+			return nil, "", fmt.Errorf("fetch GitHub stargazers page %d: %w", page, err)
+		}
+		body, readErr := io.ReadAll(resp.Body)
+		resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)